### ISSUE TITLE
If this project is not maintained, please add a notice

### ISSUE THREAD

**Issue by slhck** [pynetwork/pypcap#106]
It seems this project is not actively maintained, yet it says "now actively maintained" in the *About* field. This is a bit misleading.

Please at least give a notice in the README and project description that this project is not under active maintenance.

**Comment by hellais** [comment]
Yes you are right, I have been doing a poor job at maintaining this for a while now. The main reason has been that I don't actively use this library in any software I develop anymore. I have removed the note saying "now actively maintained" from the about field.

If anybody would like to help out with maintaining it, it would be greatly appreciated. The most important thing is to do testing for upcoming releases.

Speaking of which I merged all the PRs which seemed reasonable for an upcoming 1.3.0 release: https://github.com/pynetwork/pypcap/pull/109. It would be great to have some help in testing it.

**Comment by sreenivasulureddysura** [comment]
@hellais Give an access to other to maintain this project.

**Comment by slhck** [comment]
@sreenivasulureddysura It's not that easy — someone from the community would have to step up to do this. @hellais could probably at least change the README to ask whether someone would be willing to maintain it, if he so wishes. It's also fine if the project is marked as unmaintained so others can at least create a fork and continue maintaining it.

**Comment by hellais** [comment]
Yes I think adding a note in the readme is a great suggestion, I have done so over here: https://github.com/pynetwork/pypcap/commit/1f1b67cde5b9380f5c0a4743e0d9fb31adf604fe.

If somebody would like to step up and lead the maintenance efforts I am more than happy to assist in facilitating that process.

**Comment by pythonhacker** [comment]
@hellais - I returned to this library when I was started porting one of my early hobby projects -> https://github.com/pythonhacker/pyscanlogd to Python3 yesterday. This was written and open sourced during my days as a network security engineer at McAfee. I have the background and interest in maintaining this. Let me know what it entails if you think the credentials are good. Thank you.
